packages/core-modules/contracts/storage/GlobalStorage.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract GlobalStorage {
```

Comment:
Is there a chance that GlobalStorage could be used for something real later on? To account for that remote possibility, would it be better to name this guy `SampleStorage.sol`?

Reply:
this is a good point, for example OZ has something similar i.e. storage that can be either address, unit etc, depending on the casting